sparql/sparql12/syntax-triple-terms-negative/manifest.ttl

```diff
@@ -0,0 +1,455 @@
+@prefix :       <https://w3c.github.io/rdf-tests/sparql/sparql12/syntax-triple-terms-negative/manifest#> .
+@prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix qt:     <http://www.w3.org/2001/sw/DataAccess/tests/test-query#> .
+@prefix dawgt:  <http://www.w3.org/2001/sw/DataAccess/tests/test-dawg#> .
+
+<>  rdf:type mf:Manifest ;
+    rdfs:label "Negative Syntax Triple Terms" ;
+    rdfs:comment "Negative syntax tests for triple terms in SPARQL 1.2" ;
+    mf:entries
+    (
+:alternate-path-anonreifier
+:alternate-path-reifier
+:alternate-path-tripleterm
+:annotated-anonreifier-alternate-path
+:annotated-anonreifier-list-predicate
+:annotated-anonreifier-oneOrMore
+:annotated-anonreifier-optional
+:annotated-anonreifier-path-on-start
+:annotated-anonreifier-path
+:annotated-anonreifier-variable-path
+:annotated-reifier-list-predicate
+:annotated-reifier-oneOrMore
+:annotated-reifier-optional
+:annotated-reifier-path-on-start
+:annotated-reifier-path
+:annotated-reifier-reifier-alternate-path
+:annotated-reifier-variable-path
+:basic-tripleterm-subject
+:bind-anonreified
+:bind-reified
+:bindbnode-anonreifier
+:bindbnode-reifier
+:bindbnode-tripleterm
+:bnode-predicate-anonreifier
+:bnode-predicate-reifier
+:bnode-predicate-tripleterm
+:compound-tripleterm-subject
+:list-anonreifier-01
+:list-anonreifier-02
+:list-reifier-01
+:list-reifier-02
+:list-tripleterm-01
+:list-tripleterm-02
+:nested-annotated-path-anonreifier
+:nested-annotated-path-reifier
+:nested-tripleterm-subject
+:quoted-list-object-anonreifier
+:quoted-list-object-reifier
+:quoted-list-object-tripleterm
+:quoted-list-predicate-anonreifier
+:quoted-list-predicate-reifier
+:quoted-list-predicate-tripleterm
+:quoted-list-subject-anonreifier
+:quoted-list-subject-reifier
+:quoted-list-subject-tripleterm
+:quoted-path-anonreifier
+:quoted-path-bind-anonreifier
+:quoted-path-bind-reifier
+:quoted-path-bind-tripleterm
+:quoted-path-reifier
+:quoted-path-tripleterm
+:quoted-variable-path-anonreifier
+:quoted-variable-path-reifier
+:quoted-variable-path-tripleterm
+:subject-tripleterm
+:syntax-update-anonreifier-01
+:syntax-update-anonreifier-02
+:tripleterm-separate-01
+:tripleterm-separate-02
+:tripleterm-separate-03
+:tripleterm-separate-04
+:tripleterm-separate-05
+:tripleterm-separate-06
+
+) .
+
+:alternate-path-anonreifier rdf:type   mf:NegativeSyntaxTest ;
+   dawgt:approval dawgt:Proposed ;
+   mf:name        "alternate-path-anonreifier.rq" ;
+   mf:specVersion "sparql-1.2";
```

Comment:
We have IRIs for the different SPARQL language versions (via the [service description spec](https://www.w3.org/TR/sparql11-service-description/#instances)). I'd personally prefer if we used those instead of literals here. Thoughts?

Reply:
> We have IRIs for the different SPARQL language versions (via the [service description spec](https://www.w3.org/TR/sparql11-service-description/#instances)). I'd personally prefer if we used those instead of literals here. Thoughts?

@kasei Would you be OK with raising that as separate issue, either one it's own or one to cover all manifest vocabulary changes? This is to let the test material go in - it's big PR.

Reply:
Yes, totally fine if that's handled later.